src/server/logging/log_interceptor.js

```diff
@@ -0,0 +1,47 @@
+import Stream from 'stream';
+import { get, isEqual, sortBy } from 'lodash';
+
+function doTagsMatch(event, tags) {
+  return isEqual(sortBy(get(event, 'tags')), sortBy(tags));
```

Comment:
It just doesn't seem necessary to add the additional overhead while limiting control. Should the tag order change in Hapi we can always update them. In the future if we want to be less greedy with the matching we can then sort.